pages/conference-day.tsx

```diff
@@ -0,0 +1,233 @@
+
+const ConferenceDayPage: NextPage<ConferencePageProps> = ({ sessions }) => {
+  const { conference, dates } = useConfig()
+  if (dates.IsInProgress) {
+    // dummy
+  }
+
+  const { currentSessionGroup } = useSessionGroups(sessions)
+
+  if (currentSessionGroup && currentSessionGroup.sessions.length > 0) {
+    // NB: This is quite brittle, as it assumes that the list of current sessions is returned in exactly the same order as the definition of rooms
+    currentSessionGroup.sessions.map(function (session: Session, i) {
+      const presenters = []
+      session.Presenters.map((p) => {
+        presenters.push(p.Name)
+      })
+      if (!roomLocations.features[i]) return
+      roomLocations.features[i].properties.currentEvent = {
+        eventId: session.Id,
+        eventName: session.Title,
+        eventPresenters: presenters.join(', '),
+      }
+    })
+  }
+
+  return (
+    <Main title="Conference Day" description="Conference day information">
+      <h1>
+        {conference.Name} {format(conference.Date, 'y')} Conference Day Information
+      </h1>
+      {conference.Handbook && (
+        <p>
+          You can also download much of the information on this page as a PDF:&nbsp;
+          <a className="btn btn-pdf" href={'/static/docs/' + conference.Handbook}>
+            Download handbook (PDF)
+          </a>
+        </p>
+      )}
+      <h2>Getting There</h2>
+      <Text>
+        Our <SafeLink href="/venue">Venue</SafeLink> page has more information on how to get to the conference.
+      </Text>
+      <h2>Agenda</h2>
+      <Text>
+        {conference.Name} {format(conference.Date, 'y')} consists of nine tracks, which will be run out of the Riverside
+        Theatre, and Meeting Rooms 1 through to 8. The Keynote and Locknote will take place in the Riverside Theatre.
+        You can <SafeLink href="/agenda">view the full agenda</SafeLink> online.
+      </Text>
+      <Text>
+        Rooms will be clearly marked on the day, as well as screens outside the rooms indicating upcoming sessions.
+        Rooms have no particular theme, attend whichever sessions most interest you at each time slot. Note: We have a
+        mix of 45 minute and 20 minute sessions. If you wish to move around while sessions are in progress then feel
+        free to do so, however take care to be respectful towards our presenters.
+      </Text>
+      <h2>Around the Venue</h2>
+      <Text>
+        We are located on level 2 near the Riverside Theatre through to the Northern and Southern Foyers. We also have a
+        treasure hunt running - get your checklist from the DDD Help Desk and complete the hunt for your chance to win a
+        prize at the end of the day!
+      </Text>
+      <div style={{ zIndex: 200 }}>
+        <VenueMap roomLocationData={roomLocations} />
+      </div>
+
+      <h3>Toilets</h3>
+      <Text>
+        The above floorplan shows where the toilets are located on level 2. They will be "Access all toilets", which
```

Comment:
This has probably come from another page so more people have had eyes on it, but I would have thought "all access toilets", rather than the other way around 🤷

Reply:
That wording has actually been in the handbook since 2018, so I haven't changed it. I've queried the Engagement team about it.